features/FEATURE_BLE/ble/gap/Gap.h

```diff
@@ -492,6 +492,37 @@ class Gap {
         {
         }
 
+        /**
+         * Function invoked when the connections changes the ATT_MTU which controls
+         * the maximum size of an attribute that can be read in a single L2CAP packet
+         * which might be fragmented across multiple packets.
+         *
+         * @param connectionHandle The handle of the connection that changed the size.
+         * @param attMtuSize
+         */
+        virtual void onAttMtuChanged(
```

Comment:
This function is related to ATT/GATT; I wonder if it is the right place for it or if we should have an event handler in the gatt client and the gatt server.

Reply:
mtu event is common to both client and server so I think it's OK to have it in GAP since gap is required to be there anyway